docs/src/content/en/docs/auth/jwt.mdx

```diff
@@ -11,8 +11,10 @@ The `MastraJwtAuth` class provides a lightweight authentication mechanism for Ma
 
 ## Installation
 
-```bash copy
-npm install @mastra/auth
+Before you can use the `MastraJwtAuth` class you have to install the `@mastra/auth` package.
+
+```bash npm2yarn copy
```

Comment:
I'm tyrying not to include package manager options, as it leads to inconsistencies. The only place we really want to call out different providers is in the getting started sections. We have another ticket for a proper component to handle this so it includes all providers. But it's low priority. For now just use bash and leave it as npm install...